### PR TITLE
Update .travis/ scripts

### PR DESCRIPTION
**Pull request by notmessenger**
# Overview

## Does this PR close an existing issue?
No

## Summary
Provide a general summary of the issue addressed in the title above

## Issue Number(s)
n/a

## Checklist
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have evaluated if the _README.md_ documentation needs to be updated
* [ ] I have evaluated if the _/tests/dummy/_ app needs to be modified
* [ ] I have evaluated if DocBlock headers needed to be added or updated
* [ ] I have verified that lint and tests pass locally with my changes
* [ ] If a fork of a dependent package had to be made to address the issue this PR closes:
  * [ ] I noted in the fork's _README.md_ the reason the fork was created
  * [ ] I have opened an upstream issue detailing what was deficient about the dependency
  * [ ] I have opened an upstream PR addressing this deficiency
  * [ ] I have opened an issue in this repository to track this PR and schedule the removal of the usage of the fork

# Semver

**This project uses [semver](http://semver.org), please check the scope of this PR:**

- [x] #none#
- [ ] #patch#
- [ ] #minor#
- [ ] #major#

Examples:
* **NONE**
  * _README.md_ changes
  * test additions
  * changes to files that are not used by a consuming application (_.travis.yml_, _.gitignore_, etc)
* **PATCH**
  * backwards-compatible bug fix
    * nothing about how to use the code has changed
    * nothing about the outcome of the code has changed (though it likely corrected it)
  * changes to demo app (_/tests/dummy/_)
* **MINOR**
  * adding functionality in a backwards-compatible manner
    * nothing about how used to use the code has changed but using it in a new way will do new things
    * nothing about the outcome of the code has changed without having to first use it in a new way
    * addition of new CSS selectors
    * addition of new `ember-hook` selectors
* **MAJOR**
  * incompatible API change
    * using the code how used to will cease working
    * using the code how used to will have a different outcome
    * any changes to CSS selector names
    * any removal of CSS selectors
    * any changes to `ember-hook` selectors
    * possibly changes to test helpers (depends on the changes made)
  * any changes to the **_dependencies_** entry in the _package.json_ file

# CHANGELOG

* Update .travis/ scripts to reflect changes that should have been made in previous PR

### CODE REVIEW

File: .travis/maybe-publish-coverage.sh [hunk 1]
```diff
@@ -4,19 +4,19 @@ source $(npm root -g)/pr-bumper/.travis/is-bump-commit.sh
 
 if isBumpCommit
 then
-  echo "Skipping coverage publish for version bump commit"
+  echo "Skipping pr-bumper for version bump commit"
```
Comment (juwara0) [inline .travis/maybe-publish-coverage.sh:7]: I kept this line the same in the frost-core update: https://github.com/ciena-frost/ember-frost-core/pull/546/files#diff-872972d91442cb5b89f7fd8bebcf8a3eL7

File: .travis/maybe-bump-version.sh [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 then
-  echo "Skipping version bump for TRAVIS_NODE_VERSION ${TRAVIS_NODE_VERSION}"
+  echo "Skipping pr-bumper for TRAVIS_NODE_VERSION [${TRAVIS_NODE_VERSION}]"
   exit 0
 fi
 
 if [ "$EMBER_TRY_SCENARIO" != "ember-default" ]
 then
-  echo "Skipping version bump for EMBER_TRY_SCENARIO ${EMBER_TRY_SCENARIO}"
+  echo "Skipping pr-bumper for EMBER_TRY_SCENARIO [${EMBER_TRY_SCENARIO}]"
```
Comment (juwara0) [inline .travis/maybe-bump-version.sh:11]: This line reads differently from: https://github.com/ciena-frost/ember-frost-core/pull/546/files#diff-3aac347fbc380ab603b6957f61185df8R11

File: .travis/maybe-publish-coverage.sh [hunk 3]
```diff
@@ -4,19 +4,19 @@ source $(npm root -g)/pr-bumper/.travis/is-bump-commit.sh
...
 then
-  echo "Skipping coverage publish for TRAVIS_NODE_VERSION ${TRAVIS_NODE_VERSION}"
+  echo "Skipping pr-bumper for TRAVIS_NODE_VERSION [${TRAVIS_NODE_VERSION}]"
   exit 0
 fi
 
 if [ "$EMBER_TRY_SCENARIO" != "ember-default" ]
 then
-  echo "Skipping coverage publish for EMBER_TRY_SCENARIO ${EMBER_TRY_SCENARIO}"
+  echo "Skipping pr-bumper for EMBER_TRY_SCENARIO [${EMBER_TRY_SCENARIO}]"
```
Comment (juwara0) [inline .travis/maybe-publish-coverage.sh:19]: https://github.com/ciena-frost/ember-frost-core/pull/546/files#diff-872972d91442cb5b89f7fd8bebcf8a3eR19

File: .travis/maybe-publish-coverage.sh (hunk 1 above)
Comment (notmessenger) [reply .travis/maybe-publish-coverage.sh:7]: I know - https://github.com/ciena-frost/ember-frost-core/pull/547 changes it there as well

File: .travis/maybe-bump-version.sh (hunk 2 above)
Comment (notmessenger) [reply .travis/maybe-bump-version.sh:11]: I know - https://github.com/ciena-frost/ember-frost-core/pull/547 changes it there as well

File: .travis/maybe-publish-coverage.sh (hunk 3 above)
Comment (notmessenger) [reply .travis/maybe-publish-coverage.sh:19]: I know - https://github.com/ciena-frost/ember-frost-core/pull/547 changes it there as well

**Pull request closed without merging**